SmartCar/Arduino/sketch_main/sketch_main.ino

```diff
@@ -114,10 +207,11 @@ void loop() {
         Serial.print(range_string);
         lastDistance = range;
         distance_Delay = 0;
-    }
+    }   
+  }
 
     //Allows the car to drive backwards if an obstacle is in from of the car
-    if (velocity < 90){
+    if (velocity < 90 && !motorActivation){
```

Comment:
Shouldnt be a not motorActivation since then it wont do this while the motor is on